BZ1851483: Preparing to reinstall a cluster on bare-metal (#46137)
openshift/openshift-docs: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc

// Module included in the following assemblies:
// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]

> = Preparing to reinstall a cluster on bare-metal

Author: Done.

Before you reinstall a cluster on bare-metal, you must perform cleanup operations.

.Procedure

> . Remove or reformat the disks for the bootstrap, master, and worker nodes. If you are working in a hypervisor environment, you will need to re-add any disks you removed. You are now starting with clean disks for the bootstrap, master, and worker nodes.

Reviewer: ```suggestion
. Remove or reformat the disks for the bootstrap, <control-plane>, and worker nodes. If you are working in a hypervisor environment, you must add any disks you removed. You are now starting with clean disks for the bootstrap, <control-plane>, and worker nodes.
```

- I think that the last sentence can be cut. It doesn't add anything, and it is one more thing to wrangle with the conscious language issue mentioned in the comment above. 
- The term _re-add_ is not in the Merriam-Webster dictionary. I could not find guidance on the term in the SSG or ISG.
- [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) advises against using future tense.

Author: Done.

> . Remove or reformat the disks for the bootstrap, master, and worker nodes. If you are working in a hypervisor environment, you will need to re-add any disks you removed. You are now starting with clean disks for the bootstrap, master, and worker nodes.

Author: Resolved for 4.9 onwards.
Will create separate PR for 4.8 and earlier.

> . Remove or reformat the disks for the bootstrap, master, and worker nodes. If you are working in a hypervisor environment, you will need to re-add any disks you removed. You are now starting with clean disks for the bootstrap, master, and worker nodes.

Author: Sorry - realized I need separate PRs for 4.6, 4.7, and 4.8.

> . Delete the artifacts that the previous installation generated:
> [source,terminal]
> ----
> $ cd ; /bin/rm -rf auth/ bootstrap.ign master.ign worker.ign metadata.json .openshift_install.log .openshift_install_state.json
> ----

Reviewer: ```suggestion
. Delete the artifacts that the previous installation generated:
+
[source,terminal]
----
$ cd ; /bin/rm -rf auth/ bootstrap.ign master.ign worker.ign metadata.json \
.openshift_install.log .openshift_install_state.json
----
```
Refer to the sections on [lists](https://github.com/openshift/openshift-docs/blob/f6e9d0618b5498bd459e47ed8932a76b747a83cb/contributing_to_docs/doc_guidelines.adoc#lists) and [breaking up long lines](https://github.com/openshift/openshift-docs/blob/f6e9d0618b5498bd459e47ed8932a76b747a83cb/contributing_to_docs/doc_guidelines.adoc#lists) in the OSDOCS guidelines.

Author: Done. Will revise standards.

> [start=3]

Reviewer: By using the  `+` you should be able to delete `[start=3]` here.

Author: Done.

> . Generate new manifests and Ignition config files. Refer to the topic “Creating the Kubernetes manifest and Ignition config files" in Additional Resources, below, for further information.

Reviewer: ```suggestion
. Generate new manifests and Ignition config files. See “Creating the Kubernetes manifest and Ignition config files" for more information.
```

Author: Done.

// Module included in the following assemblies:
// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]

= Preparing to reinstall a cluster on bare metal
Before you reinstall a cluster on bare metal, you must perform cleanup operations.

.Procedure
. Remove or reformat the disks for the bootstrap, <control-plane>, and worker nodes. If you are working in a hypervisor environment, you must add any disks you removed.
. Delete the artifacts that the previous installation generated:

> 

Reviewer 2: ```suggestion
+
```

Attaching the command to the step is more future-proof than using `[start=3]`.

Author: Added + to topic.
